src/convert.js

```diff
@@ -10,15 +10,15 @@ export function convertToJson (n3String, jsonCallback) {
   })
   asyncLib.waterfall([
     function (callback) {
-      n3Parser.parse(n3String, callback)
-    },
-    function (triple, prefix, callback) {
-      if (triple !== null) {
-        n3Writer.addTriple(triple)
-      }
-      if (typeof callback === 'function') {
-        n3Writer.end(callback)
-      }
+      n3Parser.parse(n3String, function (error, quad, prefixes) {
+        if (error) {
+          throw error;
```

Comment:
Actually we want to `callback(error)` here.

Reply:
yep, [async docs](https://caolan.github.io/async/v3/docs.html#waterfall) support that. `throw error` throws all the way out of the call to `waterfall` while `callback(error)` calls the last function with whatever was thrown.